parsexml.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+#Author:        Sasan Bahadaran
+#Date:          2/29/16
+#Organization:  Commerce Data Service
+#Description:   This script crawls the files directory and finds the metadata
+#xml file for each set of pdf files.  It then gets the document id from
+#the metadata and finds the associated extracted pdf text and adds it to the
+#structure of the document, then transforms the document to json and saves it
+#to a file.
+
+import sys, json, xmltodict, os, logging, time, argparse, glob
+from datetime import datetime
+
+#this function contains the code for parsing the xml file
+#and writing the results out to a json file
+def processFile(fname,overwrite):
+    try:
+        with open(fname) as fd:
+            fn = os.path.splitext(fname)[0]+'.json'
```

Comment:
I would extract changing the extension into a method like

    def change_ext(fname, ext):
        return '.'.join(os.path.splitext(fname)[0], ext)